src/cmake_config.h.in

```diff
@@ -4,7 +4,7 @@
 #define CMAKE_CONFIG_H
 
 #define PROJECT_NAME "@PROJECT_NAME@"
-#define PROJECT_NAME_LOWER "@PROJECT_NAME_LOWER@"
+#define PROJECT_NAME_C "@PROJECT_NAME_CAPITALIZED@"
```

Comment:
`_C` suggests that it's a C string (`const char *`), and implies that `PROJECT_NAME` is a C++ string (`std::string`).